web/lib/context/Settings/index.jsx

```diff
@@ -0,0 +1,27 @@
+import { createContext, useState, useEffect } from 'react'
+import { getExampleSpeechSettings } from '../../storage'
+
+export const SettingsContext = createContext()
+
+export const SettingsProvider = ({ children }) => {
+  const [showSpeechForExamples, setShowSpeechForExamples] = useState(false)
+
+  const toggleShowExampleSpeech = (value) => {
+    getExampleSpeechSettings().set(value)
+    setShowSpeechForExamples(value)
+  }
+
+  useEffect(() => {
+    setShowSpeechForExamples((prevState) => ({
+      ...prevState,
+      showSpeechForExamples: getExampleSpeechSettings().get(),
+    }))
+  }, [])
```

Comment:
Leftover from previous implementation? Shouldn't this be:

```jsx
setShowSpeechForExamples(getExampleSpeechSettings().get())
```

Actually we can write as
```jsx
const [showSpeechForExamples, setShowSpeechForExamples] = useState(getExampleSpeechSettings().get())
```

And remove this effect at once.

Reply:
Now I remember reason for writing in this way: `useState(getExampleSpeechSettings().get())` will resolve in `window is undefined` error.